### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Import ExtensionModify Columns_Keyboard: On tab key navigation to each selected checkbox the focus on it is not visible.

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4174]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS;-A11yTCS;-A11ySev1;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG2.4.7;-Benchmark;-Keyboard;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW use the right-click key on the keyboard on it and select import wizard.
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Tab to cell column 3 row1 while the checkbox is selected.

**Actual:**
On navigating to column 3 row 1 checkbox while it is selected the tab focus on it is not visible.
Same is observed for checkbox in column 4 row1 and in row2 column 3 and 4

**Expected:**
On navigating to the checkbox while it is selected the focus should be visible on tab navigation.

**User Impact:**
The keyboard users will not know where the tab navigation is moving .

**Attachment:**
